src/main/java/com/Service/MyUserDetailService.java

```diff
@@ -0,0 +1,24 @@
+package com.Service;
+
+import com.Model.UserDetail;
+import com.Repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
not sure this one should be a @Service either,.. you're just interacting with the entities.

Reply:
this service is not needed.